scripts/generate_dashboard.py

```diff
@@ -2,13 +2,20 @@
 import json
 import os
 
+def make_progress_badge(ratio):
+    percent = f"{ratio:.1%}"
+    return f'<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="100" height="20" role="img" aria-label="Progress: {percent}"><title>Progress: {percent}</title><linearGradient id="s" x2="0" y2="100%"><stop offset="0" stop-color="#bbb" stop-opacity=".1"/><stop offset="1" stop-opacity=".1"/></linearGradient><clipPath id="r"><rect width="100" height="20" rx="3" fill="#fff"/></clipPath><g clip-path="url(#r)"><rect width="59" height="20" fill="#555"/><rect x="59" width="41" height="20" fill="#007ec6"/><rect width="100" height="20" fill="url(#s)"/></g><g fill="#fff" text-anchor="middle" font-family="Verdana,Geneva,DejaVu Sans,sans-serif" text-rendering="geometricPrecision" font-size="110"><text aria-hidden="true" x="305" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="490">Progress</text><text x="305" y="140" transform="scale(.1)" fill="#fff" textLength="490">Progress</text><text aria-hidden="true" x="790" y="150" fill="#010101" fill-opacity=".3" transform="scale(.1)" textLength="330">{percent}</text><text x="790" y="140" transform="scale(.1)" fill="#fff" textLength="330">{percent}</text></g></svg>'
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="generate the dashboard markdown")
     parser.add_argument("hist_file",
                         help="json file containing output of `lake exe extract_implications outcomes --hist")
     parser.add_argument("--out_file",
                         default="home_page/dashboard/index.md",
                         help="markdown file to pass to jekyll")
+    parser.add_argument("--badge_file",
+                        default="home_page/dashboard/progress_badge.svg",
```

Comment:
For reviewers - I'm not very familiar with Jekyll. Is this the right place to put it if we want the file to end up at
https://teorth.github.io/equational_theories/dashboard/progress_badge.svg
?

Reply:
Sorry, I didn't see this question when I approved the PR.  I guess we will see if it works once it is deployed...

Reply:
Looks like it worked.
We can add the line to the section near the beginning of the readme, I'll open a PR later today if nobody gets to it first.